adafruit_display_shapes/polygon.py

```diff
@@ -121,13 +138,13 @@ def _line(self, x0, y0, x1, y1, color):
     # pylint: enable=invalid-name, too-many-locals, too-many-branches
 
     @property
-    def outline(self):
+    def outline(self) -> int:
         """The outline of the polygon. Can be a hex value for a color or
         ``None`` for no outline."""
         return self._palette[1]
 
     @outline.setter
-    def outline(self, color):
+    def outline(self, color: int) -> None:
```

Comment:
same for the color passed in to the setter here, it can be `None` for no outline so for this argument it would be like this:
```
def outline(self, color: Optional[int]) -> None:
```